airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/models/AttemptNormalizationStatus.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.persistence.job.models;
+
+import java.util.Optional;
+
+public class AttemptNormalizationStatus {
```

Comment:
Could this be replaced with a `record class`?  
Should `normalizationFailed` be a `boolean` stead, forcing a true/false to be provided?